Tests/SwiftGenTests/ConfigTests.swift

```diff
@@ -0,0 +1,267 @@
+//
+// SwiftGen
+// Copyright (c) 2015 Olivier Halligon
+// MIT Licence
+//
+
+import XCTest
+import PathKit
+
+class ConfigTests: XCTestCase {
+
+  func testReadConfigWithParams() throws {
+    guard let path = Bundle(for: type(of: self)).path(forResource: "config-with-params", ofType: "yml") else {
+      fatalError("Fixture not found")
+    }
+    let file = Path(path)
+    do {
+      let config = try Config(file: file)
+      XCTAssertEqual(config.outputDir, "Common/Generated")
```

Comment:
Noticed this really cool library by merowing in the slack recently:
https://github.com/krzysztofzablocki/Difference

Could be useful here, and if not, it's definitely useful for the context tests.

Reply:
Yeah, I thought about it at some point. For comparing the generated files in SwiftGenKit it's no good, as we're comparing basically long texts there and not structures, but for here where we compare `Config` instances, it might be worth it 🤔